documentation/content/en/books/handbook/introduction/_index.adoc

```diff
@@ -75,7 +75,7 @@ Its particular strengths are:
 * _Extensive security features_, from the Mandatory Access Control framework to Capsicum capability and sandbox mechanisms.
 * _Over 30 thousand prebuilt packages_ for all supported architectures, and the Ports Collection which makes it easy to build customized ones.
 * _Documentation_ - in addition to the Handbook and books from different authors that cover topics ranging from system administration to kernel internals, there are also the man:man[1] pages, not only for userspace daemons, utilities, and configuration files, but also for kernel driver APIs (section 9) and individual drivers (section 4).
-* _Simple and consistent repository structure and build system_ - FreeBSD uses a single repository for all of its components, both kernel and userspace. This, along with a unified and easy to customize build system and a well thought-out development process makes it easy to integrate FreeBSD with build infrastructure for own product.
+* _Simple and consistent repository structure and build system_ - FreeBSD uses a single repository for all of its components, both kernel and userspace. This, along with a unified and easy to customize build system and a well thought-out development process makes it easy to integrate FreeBSD with build infrastructure for your own product.
```

Comment:
Good catch, but the FreeBSD style guide recommends against using "you". I'd suggest instead something like "infrastructure for third-party or derived products".